Cargo.toml

```diff
@@ -7,4 +7,4 @@ edition = "2021"
 
 [dependencies]
 libc = "0.2"
-subprocess = "0.2"
+subprocess = { version = "= 0.2.9" }
```

Comment:
it would have been sufficient to do: `subprocess = "0.2.9"`

Reply:
Are you sure?  In https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html we read about the version "0.1.12":

> Although it looks like a specific version of the time crate, it actually specifies a range of versions and allows [SemVer](https://semver.org/) compatible updates. An update is allowed if the new version number does not modify the left-most non-zero number in the major, minor, patch grouping. In this case, if we ran `cargo update time`, cargo should update us to version 0.1.13 if it is the latest 0.1.z release, but would not update us to 0.2.0.
...
> This compatibility convention is different from SemVer in the way it treats versions before 1.0.0. While SemVer says there is no compatibility before 1.0.0, Cargo considers 0.x.y to be compatible with 0.x.z, where y ≥ z and x > 0.

What I was trying to express here was that `cargo update` should not be allowed to update this crate, because we do not trust it, and since Cargo does Weird Things before version 1.0.0, my understanding was that the "=" is required in this case.  Had the module been at 1.0.0 or higher you would indeed be right.

Reply:
Ah! Thanks for explanation! I visually missed the "=" inside the quotes.